articles/azure-government/documentation-government-services-securityandidentity.md

```diff
@@ -30,6 +30,19 @@ The following information identifies the Azure Government boundary for Azure Key
 | --- | --- |
 | All data encrypted with an Azure Key Vault key may contain Regulated/controlled data. |Azure Key Vault metadata is not permitted to contain export-controlled data. This metadata includes all configuration data entered when creating and maintaining your Key Vault.  Do not enter Regulated/controlled data into the following fields: **Resource group names, Key Vault names, Subscription name** |
 
+## Azure Active Directory Premium
```

Comment:
You should also add an entry for Azure AD Premium in the [Available Services](https://docs.microsoft.com/azure/azure-government/documentation-government-services#security--identitydocumentation-government-services-securityandidentitymd) doc.